ddsrouter_core/src/cpp/core/DDSRouterImpl.cpp

```diff
@@ -334,6 +336,14 @@ void DDSRouterImpl::discovered_topic_(
     }
 }
 
+void DDSRouterImpl::discovered_endpoint_(
+        const Endpoint& endpoint) noexcept
+{
+    logInfo(DDSROUTER, "Discovered endpoint: " << endpoint << ".");
```

Comment:
I changed it to Debug in order to avoid having two almost identical consecutive logInfos, but I will keep the message general. The proposed message is specific to the discovery functionality.

Reply:
This log refers to DDS Router behaviour, so it should specify that this is happening in DDS Router. If you want to log something in DiscoveryDatabase, do it in that class implementation.